app/serializers/institution_profile_serializer.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+class InstitutionProfileSerializer < ActiveModel::Serializer
+  include OptionalSerializedAttributes
+
+  attribute :institution
+  attribute :facility_code
+  attribute :institution_type_name, key: :type
+  attribute :city
+  attribute :state
+  attribute :zip
+  attribute :country
+  attribute :bah
+  attribute :cross
+  attribute :ope
+  attribute :pred_degree_awarded
+  attribute :locale
+  attribute :gibill, key: :student_count
```

Comment:
For consistency, I have to keep field names the same as the existing Data.csv as used by the VA EDU folks.